luxonis_train/models/luxonis_lightning.py

```diff
@@ -268,6 +273,16 @@ def __init__(
 
         self.load_checkpoint(self.cfg.model.weights)
 
+        if self.cfg.trainer.training_strategy.params:
```

Comment:
Check if params are specified to determine if training strategy is specified feels a bit weird. Should the whole training strategy section in config rather be None by default and then if used we make `name` a required parameter like we have for `CallbackConfig`? CC: @kozlov721 also for thoughts

Reply:
Yeah, I completely agree—that approach would make much more sense. Initially, I followed the logic of using predefined Adam and constantLR for the optimizer and scheduler, applying the same idea to the strategy. However, that feels unintuitive and should definitely be changed. Also, @kozlov721, what’s your take on this?

Reply:
Now I see what I was doing wrong (when getting that error). I'll fix it...